ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -160,7 +160,30 @@ class _ConnectionKey(NamedTuple):
     instrument_type_id: str
 
 
-class BaseReservation(abc.ABC):
+class _BaseSessionContainer(abc.ABC):
+    """Manages Session Management Client."""
```

Comment:
At the moment, maybe this is a better comment to describe the class.

```suggestion
    """Contains session manager and related properties."""
```

Reply:
yeah, docstrings are not in title case

Reply:
Updated to `"""Contains session management client and related properties."""`